src/main/java/servlets/VendorServlet.java

```diff
@@ -33,12 +44,68 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     JsonConverter converter = new JsonConverter();
     String configMap = converter.getConfigMap();
-    response.setContentType("text/html;");
+    response.setContentType(CONTENT_TYPE);
     response.getWriter().println(configMap);
   }
 
-  @Override
-  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // DELETE method --> deletes vendor configuration directory
+  /** Delete the desired Vendor from the fileset and spreadsheets if posible.*/
+  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {  
+    String vendorID = request.getParameter(VENDOR_ID_PARAM);
+    String deleteStatus = "success";
+
+    // Delete the file from the filesystem and update the sheets.
+    try {
+      deleteVendorFile(vendorID);
+      updateSheets(request);    
```

Comment:
Using a string to indicate success/failure doesn't feel super robust, but I'm not sure of a better alternative off the top of my head? What alternatives did you consider?

Reply:
Maybe an alert or a snackbar could be implemented as confirmation?